### ISSUE TITLE
Size of Bank + warband + bags on screen

### ISSUE THREAD

**Issue by 1Holy-Z** [Jaliborc/Bagnon#1926]
Thank you for updating this great addon btw!

**Is your feature request related to a problem? If so, please describe.**
I have not even purchased all of the warband bank tabs and I can barely fit my bags and bank on screen together at a somewhat reasonable/readable size.  This will be a bigger problem with a fully upgraded warband bank.

**Describe the solution you'd like.**
perhaps a tab to open warband bank in a separate tab while in normal bank tab like with the blizz UI.

**Comment by Clarie91** [comment]
I agree too! I already have the bank window scaled down to 84% to fit three warband tab into the window and it doesn't include the reagent bank as it has only Dragonflight things like Artisan's Mettle and such. A separate window for warband would be very much appreciated.

**Comment by Licen-it** [comment]
I was about to request the same feature...i can barely fit into my screen too

**Comment by Dreamcatcher35** [comment]
I agree too! That would be awesome.

**Comment by Lusaphor** [comment]
This would be amazing. I think making the warbank a seperate window from the bank, would allow for unique customizations and other feature for it, and a much better ease of use in conjunction while use the bank.

**Comment by hyperrex** [comment]
Definitely would appreciate some kind of UI that lets me toggle bank/warbank visibility. It's so big now, the visual noise + reduced size of the tiles to fit is making Bagnon not as useful as it used to be. Way too much scanning IMO.
